### PR TITLE
#11464 ForeignKey name is not provided 

### PR DESCRIPTION
**Pull request by joaoppereira**
| Q | A |
| --- | --- |
| Is bugfix? | no |
| New feature? | yes |
| Breaks BC? | --- |
| Tests pass? | yes |
| Fixed issues | #11464 |

Added foreign key name to foreign key information.

### CODE REVIEW

**Comment by SilverFire** [comment]
The PR seems to be overseen, sorry.

**Review by SilverFire — APPROVED** [review]
Looks good to me. However, it looks like a BC break.

@yiisoft/core-developers

File: framework/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Yii Framework 2 Change Log
 
 - no changes in this release.
 
+- Bug #11912: Fixed PostgreSQL Schema to support negative default values for integer/float/decimal columns (nsknewbie)
```
Comment (samdark) [inline framework/CHANGELOG.md]: This line looks wrong.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Merged: https://github.com/yiisoft/yii2/commit/3692941eadc3deaa0699a916a8796fd34dc96801

Thank you!

**Pull request closed without merging**